.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: ruby
+
+rvm:
+  - 2.5.0
+  - 2.6.0
+  - ruby-head
+
+dist: bionic
+
+jobs:
+  allow_failures:
+    - rvm: ruby-head
+
+env:
+  global:
+    - CC_TEST_REPORTER_ID=a3068b786ab61159b292bd4bae496bd03f98893d2016b5a73e06c185550517b2
+
+before_script:
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
+
+script:
+  - bundle exec rake code_analysis
```

Comment:
question, do we have this task in this repo? if not we might want to add it

Reply:
Added on last commit 🙌